nj-core/Cargo.toml

```diff
@@ -11,6 +11,10 @@ license = "Apache-2.0"
 [lib]
 test = false
 
+[features]
+default = ["serde_json"]
```

Comment:
This seems like something that shouldn't be a default feature, though I suppose I don't have a super strong opinion on it.

Reply:
My personal use case was that serde_json values are very common when working with json in general and node/js in particular. But if you feel like serde is not something everyone is going to use often, I can make the feature optional.

Reply:
Let's make it optional for now.

Reply:
Sounds good